internal/docker/podman.go

```diff
@@ -0,0 +1,171 @@
+package docker
+
+import (
+	"context"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"sync"
+	"time"
+
+	"github.com/anchore/stereoscope/internal/log"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/homedir"
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/ssh"
+)
+
+var (
+	makePodmanClientOnce sync.Once
+	defaultHost          = "ssh://core@localhost:63753/run/user/1000/podman/podman.sock?secure=false"
+)
+
+func getPodmanClient() (*client.Client, error) {
+	makePodmanClientOnce.Do(func() {
```

Comment:
this function presumes that podman will only be available over SSH, which is true on a mac, however, on a linux box this is not true.

Reply:
I updated it so ssh and unix path are supported, which covers linux & Mac instances.